castervoice/lib/text_manipulation_functions.py

```diff
@@ -75,7 +75,7 @@ def text_manipulation_copy(application):
     ## Pause(copy_pause_time_dict[application]).execute()
     ## selected_text = pyperclip.paste()
     
-    err, selected_text = context.read_selected_without_altering_clipboard(pause_time=copy_pause_time_dict[application])
+    err, selected_text = context.read_selected_without_altering_clipboard(same_is_okay=True, pause_time=copy_pause_time_dict[application])
```

Comment:
Eventually, we may want change this back to same_is_okay=False eventually and use it to make it try copying again (with an increased pause time) if `err` == 1 which I think means that the clipboard hasn't changed.

Reply:
Can you mention this in the "Known issues" section?

Reply:
Is the function able to distinguish between a failed copy and successfully copying the thing that's sitting on the clipboard?

Reply:
It seems that it is not able to distinguish between those based on a quick test I did. Let me know if you have a different experience.

Reply:
I think it's still better to do `same_is_ok=True` and caveat that the pauses might mess stuff up from time to time.